pegasus/forms/hoc_signup_2016.rb

```diff
@@ -0,0 +1,7 @@
+require pegasus_dir 'forms/hoc_signup_2014'
```

Comment:
We could rename the original form but I don't see why we need to? I think it's clearest to show that this was the first form and we're re-using it. Plus we'd have to update SOLR and everywhere we query 2014 stuff.